src/gui/resource.c

```diff
@@ -204,7 +204,7 @@ RESOURCE_ENTRY_PNG(png_arrow_up_orange_8px)
 RESOURCE_ENTRY_PNG(png_arrow_down_orange_8px)
 
 RESOURCE_ENTRY_PNG(png_back_32px)
-RESOURCE_ENTRY_PNG(png_serial_printing)
+RESOURCE_ENTRY_NUL()
```

Comment:
??

Reply:
Omg

Thanks for reminding me of the small issue: it won't fit the FLASH 🙈
This was outgoing issue so at some point I just ignored it and did this workaround. Now it is the time to fix it. I am gonna look at our options.

Reply:
ok, no problem